src/ocl/convolutionocl.cpp

```diff
@@ -87,24 +87,9 @@ static inline bool IsAnyBufferBF16(const TensorDescriptor& xDesc,
 }
 #endif
 
-size_t GetKernelGlobalWorkDim(const KernelInvoke& kernel, int dim)
-{
-#if(MIOPEN_BACKEND_HIP)
-    return kernel.gdims[dim];
-#else
-    return kernel.global_work_dim[dim];
-#endif
-}
+size_t GetKernelGlobalWorkDim(const KernelInvoke& kernel, int dim) { return kernel.gdims[dim]; }
```

Comment:
What is this change for?

Reply:
Changed it for updating gdims for each run, correct? @JehandadKhan

Reply:
That is correct, I updated the member names to make them uniform between both the HIP and the OpenCL backend

Reply:
Curious that they were different in the first place. Are you sure there are no side effects (usages of the old style)?